src/assets/locales/en/phovea.json

```
@@ -0,0 +1,98 @@
+{
+  "clue": {
+
+    "annotationPlaceholder": "Enter Text by Clicking (MarkDown supported)",
+    "errorMessage": "Provenance Graph with id {{graphID}} not found",
+    "exploration": "Exploration",
+    "authoring": "Authoring",
+    "presentation": "Presentation",
+
+    "provenanceMenu": {
+      "import": "Import...",
+      "importRemote": "Import Remote...",
+      "export": "Export...",
+      "new": "New...",
+      "newRemote": "New Remote...",
+      "selectFile": "Select File",
+      "upload": "Upload",
+      "selectFileToUpload": "Select File to Upload",
+      "creator": "creator:",
+      "created": "created:",
+      "description": "description:",
+      "nodes": "nodes/edges:",
+      "clone": "Clone",
+      "delete": "Delete",
+      "select": "Select",
+      "areYouSureToDelete": "Are you sure to delete: \""
```

```suggestion
      "areYouSureToDelete": "Are you sure to delete:"
```
Please avoid half quotes, as these might confusion and errors. Add the `"` to the original code or inline the whole template variable.